type-definitions/quagga.d.ts

```diff
@@ -868,6 +868,19 @@ export interface QuaggaJSConfigObject {
      */
     frequency?: number;
 
+    /**
+     * Canvas configuration options for controlling overlay and debug canvases.
+     */
+    canvas?: {
+        /**
+         * Whether to create the overlay canvas (drawingBuffer) for drawing bounding boxes
+         * and scan lines. Set to false if you don't need visual feedback and want to
+         * improve performance.
+         * @default true
+         */
+        willCreateOverlay?: boolean;
```

Comment:
i think just "createOverlay" is the correct form to use here, "will" is a future thing, and we're not saying "we will do this" we're saying "do this".  Update all instances to match.